modules/java/pages/gs-install.adoc

```diff
@@ -517,9 +537,52 @@ NOTE:  The generated war file will be at <PROJECT ROOT>/build/libs.
 +
 TIP: You can also use Tomcat’s Manager App to deploy the war file -- see https://tomcat.apache.org/tomcat-9.0-doc/manager-howto.html[Tomcat's Manager App] documentation for more detail.
 
-. To use common class loader approach to load {cbl} libraries, copy all of the {cbl} jar files in $CATALINA_BASE/lib.
+. To use common class loader approach to load Couchbase{nbsp}Lite libraries, copy all of the Couchbase{nbsp}Lite jar files in $CATALINA_BASE/lib.
 +
-NOTE: For linux platform see also -- _Using Native Libraries for Linux_ in {gs-prereqs--xref}
+NOTE: For linux platform see also -- _Using Native Libraries for Linux_ in xref:java:gs-prereqs.adoc[Prerequisites]
+
+
+[#related-content]
+== Related Content
+++++
+<div class="card-row three-column-row">
+++++
+
+[.column]
+====== {empty}
+.How to . . .
+* xref:java:gs-prereqs.adoc[Prerequisites]
+* xref:java:gs-install.adoc[Install]
+* xref:java:gs-build.adoc[Build and Run]
+
+
+.
+
+[.column]
+====== {empty}
+.Learn more . . .
+* xref:java:database.adoc[Databases]
+* xref:java:document.adoc[Documents]
+* xref:java:blob.adoc[Blobs]
+* xref:java:replication.adoc[Remote Sync Gateway]
+* xref:java:conflict.adoc[Handling Data Conflicts]
+
+.
+
+
+[.column]
+====== {empty}
+.Dive Deeper . . .
+https://forums.couchbase.com/c/mobile/14[Mobile Forum] |
+https://blog.couchbase.com/[Blog] |
+https://docs.couchbase.com/tutorials/[Tutorials]
+
+.
+
+
+++++
+</div>
+++++
 
+:page-toclevels: 2
```

Comment:
I think this is resetting the earlier override (e.g. another artefact of Antora Assembler)